components/movementsensor/replay/replay.go

```diff
@@ -28,19 +28,32 @@ import (
 const (
 	timeFormat            = time.RFC3339
 	grpcConnectionTimeout = 10 * time.Second
-	downloadTimeout       = 30 * time.Second
 	maxCacheSize          = 1000
 )
 
+type method string
+
+const (
+	linearVelocity     method = "LinearVelocity"
+	angularVelocity    method = "AngularVelocity"
+	orientation        method = "Orientation"
+	position           method = "Position"
+	compassHeading     method = "CompassHeading"
+	linearAcceleration method = "LinearAcceleration"
+)
+
 var (
 	// model is the model of a replay movement sensor.
 	model = resource.DefaultModelFamily.WithModel("replay")
 
-	// ErrEndOfDataset represents that the replay sensor has reached the end of the dataset.
-	ErrEndOfDataset = errors.New("reached end of dataset")
+	// errEndOfDataset represents that the replay sensor has reached the end of the dataset.
+	errEndOfDataset = errors.New("reached end of dataset")
+
+	// errCloudConnectionFailure represents that the attempt to connect to the cloud failed.
+	errCloudConnectionFailure = errors.New("failure to connect to the cloud")
 
 	// methodList is a list of all the base methods possible for a movement sensor to implement.
-	methodList = []string{"Position", "Orientation", "AngularVelocity", "LinearVelocity", "LinearAcceleration", "CompassHeading"}
+	methodList = []method{linearVelocity, angularVelocity, orientation, position, compassHeading, linearAcceleration}
```

Comment:
Rearranged the order here and below to match the order of the movementsensor's properties: https://github.com/viamrobotics/rdk/blob/1bdd3047220367d80a4eafd1792bbd79f15e1441/components/movementsensor/properties.go#L7

Reply:
Changed this to match the order of the endpoints - made additional changes to other files as well to follow that same order.